### ISSUE TITLE
How to disable the line truncating?

### ISSUE THREAD

**Issue by pro-anon** [algernon/elfeed-goodies#38]
I don't like the line truncating, I prefer to use word wrapping instead.

**Comment by jeetelongname** [comment]
Line truncating in search mode?

**Comment by pro-anon** [comment]
No, when reading an entry. The lines are broken up, it automatically adds a new line at the end of each line, although the original text on the website it's just one paragraph. It's more obvious when you copy multiple lines, when you paste them you can see they are treated as separate lines not just one long line.

**Comment by pro-anon** [comment]
I think this effect is from `fill-column` or `fill` something in emacs. Is it possible to disable this filling and just wrap the lines instead, like you know, when it shows that little arrow thing on the right side of the line?

**Comment by EasternLi** [comment]
If I've got it right, `elfeed` use `shr` to render, and you need to customize `Shr Fill Text`.
